containerd-shim/process.go

```diff
@@ -75,11 +76,12 @@ func newProcess(id, bundle, runtimeName string) (*process, error) {
 	}
 	p.state = s
 	if s.Checkpoint != "" {
-		cpt, err := loadCheckpoint(bundle, s.Checkpoint)
+		cpt, err := loadCheckpoint(s.Checkpoint)
```

Comment:
Could you rename the shim's `processState::Checkpoint` to `CheckpointPath` for clarity?